bpf/lib/lb.h

```diff
@@ -48,6 +48,17 @@ struct bpf_elf_map __section_maps LB6_AFFINITY_MAP = {
 };
 #endif
 
+#ifdef ENABLE_LB_SRC_RANGE_CHECK
```

Comment:
Probably makes sense to extend `bpf/Makefile` to add this to the permutations we build-test.